### ISSUE TITLE
Depth map confusion

### ISSUE THREAD

**Issue by hsab** [vt-vl-lab/3d-photo-inpainting#25]
Hi there!

I tried converting the generated .npy depth map to tiff using PIL's `Image.fromarray`.
Upon inspection the depth map seems to represent a vertical gradient without any features from the original image. Was wondering if this is correct or I'm missing something somewhere?

![Screenshot_20200416_164408](https://user-images.githubusercontent.com/11205091/79516887-82713000-8001-11ea-9835-30c7fc9a7f1c.png)

Thank you for sharing the research. The possibilities of this are immense.

**Comment by LemonATsu** [comment]
Try normalizing the depth max before saving it as an image by:

`(depth - depth.min())/(depth.max()-depth.min())`

**Comment by hsab** [comment]
I tried that before submitting the issue, however in an image processing software.
Nevertheless, this doesn't seem to solve the issue
The following image is the result of the code below. I opted for TIFF because of 32bit support.

![Screenshot_20200416_192711](https://user-images.githubusercontent.com/11205091/79525681-58773800-8018-11ea-8146-45fe4500abe6.png)

```
import numpy
ss = numpy.load('moon.npy')
ss= (ss - ss.min())/(ss.max()-ss.min())

from PIL import image
im = Image.fromarray(ss)
im.save("depth.tiff")
```
